functests/4_latency/latency.go

```diff
@@ -144,28 +146,32 @@ var _ = Describe("[performance] Latency Test", func() {
 				Skip("no maximum latency value provided, skip buckets latency check")
 			}
 
-			latencies := extractLatencyValues("oslat", `Maximum:\t*([\s\d]*)\(us\)`, workerRTNode)
-
+			latencies := extractLatencyValues(`Maximum:\t*([\s\d]*)\(us\)`, workerRTNode)
 			// under the output of the oslat very often we have one anomaly high value, for example
 			// Maximum:    16543 15 15 14 13 12 12 13 12 12 12 12 12 12 12 12 12 (us)
 			// it still unclear if it oslat bug or the kernel one, but we definitely do not want to
 			// fail our test on it
 			var anomaly bool
-			for _, lat := range strings.Split(latencies, " ") {
+			var latenciesInt []int
+			latenciesList := strings.Split(latencies, " ")
+			for _, lat := range latenciesList {
```

Comment:
Can you please explain why do you need it? IIUC you want to be sure that you pass only over ints, but if we have some unexpected output we should improve or regex

Reply:
yes, you are right, I improved this code.